glean-core/android/src/main/java/mozilla/telemetry/glean/rust/LibGleanFFI.kt

```diff
@@ -69,7 +69,7 @@ internal interface LibGleanFFI : Library {
 
         internal var INSTANCE: LibGleanFFI = try {
             val lib = Native.load(JNA_LIBRARY_NAME, LibGleanFFI::class.java) as LibGleanFFI
-            lib.glean_enable_logging()
+            lib.glean_enable_logging(0)
```

Comment:
nit: can we add a comment explaining why we're passing 0 here? (I know it's explained elsewhere, too)